client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -113,9 +118,9 @@ class EditorRevisionsList extends PureComponent {
 			comparisons,
 			postId,
 			revisions,
+			isRequestingRevisions,
 			selectedRevisionId,
 			siteId,
-			newRevisionIds,
 		} = this.props;
 		const classes = classNames( 'editor-revisions-list', {
```

Comment:
Somewhat unrelated, but I think it'd be nice to check if revisions exist early in this render method (or possibly better, in the parent `EditorRevisions` render) and return early if they don't - there's a decent amount of unnecessary calculation going on after this point if they don't exist.

Additionally, you could set it to a const (i.e. `const hasRevisions = ! isEmpty( revisions );`) and use that throughout the conditionals further down.